docs/bundles/SyliusRbacBundle/index.rst

```diff
@@ -1,7 +1,7 @@
 SyliusRbacBundle
 ================
 
-Role-Based-Access-Control for Symfony2 applications, integrated with Doctrine ORM. Managable via UI.
+Role-Based-Access-Control for Symfony applications, integrated with Doctrine ORM. Managable via UI.
```

Comment:
Ah, I always wondered what RBAC meant :) Maybe `Role Based Access Control (RBAC)` ? Don't *think* it is correct to hypenate here